app/controllers/admin/admin_controller.rb

```diff
@@ -7,4 +7,13 @@ class Admin::AdminController < ApplicationController
   def index
     render body: nil
   end
+
+  private
+
+  def preload_additional_json
+    store_preloaded(
+      "enabledPluginAdminRoutes",
+      MultiJson.dump(Discourse.plugins_sorted_by_name.map(&:admin_route).compact),
```

Comment:
Can also be a single call:

```suggestion
      MultiJson.dump(Discourse.plugins_sorted_by_name.filter_map(&:admin_route)),
```

Reply:
Damn it I always tell Sam about this then I forget about it myself :facepalm: